cloudfront-tariff-reporting.tf

```
@@ -11,7 +11,7 @@ resource "aws_cloudfront_origin_access_identity" "trade_tariff_reporting_identit
 resource "aws_cloudfront_distribution" "s3_distribution_trade_tariff_reporting" {
   origin {
     domain_name = aws_s3_bucket.this["reporting"].bucket_regional_domain_name
-    origin_id   = var.s3_origin_id
+    origin_id   = aws_s3_bucket.this["reporting"].id
```

```suggestion
    origin_id   = aws_cloudfront_origin_access_identity.trade_tariff_reporting_identity.cloudfront_access_identity_path
```

The origin access identity resource has a special output value for this purpose, but it's only documented on that resource's [documentation page](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudfront_origin_access_identity#using-with-cloudfront).

So this is all you need to change here 😄